recipes/alphacsc/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set version = "0.3" %}
+
+package:
+  name: alphacsc
+  version: {{ version }}
+
+source:
+  url: https://github.com/alphacsc/alphacsc/archive/{{ version }}.tar.gz
+  sha256: 008acfcd39e8f9b926a22535015ae09a4e6e85986d496d4b279178b396a1dda2
+  patches:
+   - patches/001-remove_sporco_files_from_package.patch
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - cython                                 # [build_platform != target_platform]
+    - numpy                                  # [build_platform != target_platform]
```

Comment:
Are you sure all of these are needed here?

Reply:
I am actually not sure of what is needed.
At the very least, `numpy` and `cython` are. `python` too to run them and the compiler.
So I will keep them for now.

Reply:
I'd suggest to try and remove all but the C compiler from this section and see if the build still works